destination/src/main/scala/quasar/plugin/sqlserver/destination/SQLServerDestination.scala

```diff
@@ -50,10 +47,10 @@ private[destination] final class SQLServerDestination[F[_]: ConcurrentEffect: Mo
 
   val destinationType = SQLServerDestinationModule.destinationType
 
-  def createSink(path: ResourcePath, columns: NonEmptyList[Column[SQLServerType]]) =
-    CsvCreateSink[F](writeMode, xa, logger, schema)(path, columns)
-
-  val sinks = NonEmptyList.one(ResultSink.create(createSink))
+  val sinks = NonEmptyList.of(
+    ResultSink.create(CsvCreateSink(writeMode, xa, logger, schema)),
+    ResultSink.upsert(CsvUpsertSink(xa, writeMode, schema, logger)),
```

Comment:
It'd be nice if we didn't have to specify both `upsert` and `append` when they don't differ. I haven't given this much thought, but could the `ResultPush` implementation use an `append` sink when present and fallback to an `upsert` sink if not?

Reply:
I thought about this too, but I'm not absolutely sure, that append and upsert sinks always have the same behaviour w/o `Delete`. It is the same right now, yes, but in general, it might differ.

Reply:
Yeah, something to keep in mind as we think about this going forward.